hoomd/md/ActiveForceCompute.cc

```diff
@@ -173,81 +216,64 @@ void ActiveForceCompute::setForces()
         {
         unsigned int tag = m_group->getMemberTag(i);
         unsigned int idx = h_rtag.data[tag];
```

Comment:
No need to use tag indirection, access the index directly.
`unsigned int idx = m_group->getMemberIndex(group_idx);`